am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/DefaultRoleSetupDataFactory.java

```diff
@@ -11,23 +11,22 @@
 import java.util.Map;
 import java.util.Set;
 
+import static uk.gov.hmcts.reform.amlib.enums.SecurityClassification.PUBLIC;
 import static uk.gov.hmcts.reform.amlib.helpers.TestConstants.RESOURCE_NAME;
 import static uk.gov.hmcts.reform.amlib.helpers.TestConstants.RESOURCE_TYPE;
 import static uk.gov.hmcts.reform.amlib.helpers.TestConstants.ROLE_NAME;
 import static uk.gov.hmcts.reform.amlib.helpers.TestConstants.ROOT_ATTRIBUTE;
 import static uk.gov.hmcts.reform.amlib.helpers.TestConstants.SERVICE_NAME;
 
+@SuppressWarnings("LineLength")
```

Comment:
As much as I would love to have it here my reserved side tells me this is not a good idea. Exceptions for a method or fragment makes sense but excluding whole class might encourage people having long lines everywhere.

Reply:
I am going to keep it. I find it unhelpful and confusing having method titles and params on different lines.